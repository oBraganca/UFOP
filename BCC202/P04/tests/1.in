36 60 84
