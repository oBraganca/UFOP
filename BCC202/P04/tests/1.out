MDC = 12
MMC = 1260
